src/bci_build/package/apache_tomcat.py

```diff
@@ -52,52 +54,47 @@ def _get_sac_supported_until(
         package_name=f"apache-tomcat-{tomcat_major}-java-{jre_version}-image"
         if os_version.is_tumbleweed
         else f"sac-apache-tomcat-{tomcat_major}-java{jre_version}-image",
+        package_list=[
+            tomcat_pkg := (
+                "tomcat"
+                if tomcat_major == _TOMCAT_VERSIONS[0]
+                else f"tomcat{tomcat_major}"
+            )
+        ]
+        + [f"java-{jre_version}-openjdk", f"java-{jre_version}-openjdk-headless"],
         os_version=os_version,
         is_latest=(
             (os_version in CAN_BE_LATEST_OS_VERSION)
             and tomcat_major == _TOMCAT_VERSIONS[-1]
             and jre_version == 22
             and os_version.is_tumbleweed
         ),
+        _oci_image_version=format_version(
+            tomcat_pkg_version := get_pkg_version(tomcat_pkg, os_version),
+            ParseVersion.PATCH,
+        ),
         version=f"{tomcat_major}-jre{jre_version}",
```

Comment:
Or we don't add the new property and instead implement this change, which should have the same consequence:
```suggestion
        version=tomcat_pkg_version,
```

Reply:
That doesn't work, as we need to add the jre suffix to make the build tag/buildversion unique across many flavors.

Also, it would change the buildversion on every minor version bump, which resets the release counter and means qa has to update their syncing and testing scripts. That seems very high overhead to me.

What we can do instead is add a version_flavor that is taken as a suffix for the tag and buildversion  if set. What do you think about that instead?

Reply:
I pushed a change that uses now a version_flavor variant but this is getting royally messy as "version" is used all over the place as a identifier and now I had to overload `version_in_uid` with a `str` to deduplicate the tomcat containers.

Reply:
actually the whole `version_flavor` mess was too complicated. now  `version` is the oci version and `tag_version` is what version used to be before. I hope this is a better approach, as it is fairly simple.